docs/source/tutorial/production.md

```diff
@@ -31,24 +31,14 @@ The entry should basically look like this:
 APOLLO_KEY=user:xz.5r134305-88p1-4840-12c1-88rc0xcxe179:E4VSTiXeFWaSSBgFWXOiSA
 ```
 
-Our key is now stored under the environment variable `APOLLO_KEY`. We now have to create a file called `apollo.config.js`. Apollo projects are configured using an `apollo.config.js` file at the root of your project. Many Apollo tools leverage your Apollo config, reducing the net amount of configuration you need to do in your project in the end.
-
-Create `apollo.config.js` with the following configuration:
-
-```js{3}
-module.exports = {
-  service: {
-    name: 'my-graph-name' // the name of your graph in Apollo Graph Manager
-  }
-};
-```
+Our key is now stored under the environment variable `APOLLO_KEY`.
 
 ### Check and publish with the Apollo CLI
 
 It's time to publish our schema to Graph Manager! First, start your server in one terminal window by running `npm start`. In another terminal window, run:
```

Comment:
"In another terminal window, run the following command, providing the name of your graph where indicated:"